content/en/user-guide/ci/github-actions/samples/index.md

```diff
@@ -0,0 +1,396 @@
+---
+title: "Sample"
+linkTitle: "Sample"
+weight: 5
+description: >
+  Localstack Sample Project in Github Actions
+aliases:
+  - /ci/github-actions/samples
+tags: ["continuous-integration", "ci", "continuous-delivery", "testing","github-actions","sample"]
+---
+
+This page containes our Quickstart project's test pipeline implemented in Github actions.
+
+# Serverless image resizer s3 lambda
+This is an app to resize images uploaded to S3 in a serverless way.
+Find code in repo [here](https://github.com/localstack-samples/sample-serverless-image-resizer-s3-lambda).
+
+## Integration testing
+A build matrix for Ubuntu and MacOS to run integration tests triggered by various ways.
+In case of failure the pipeline dumps the logs of Localstack and preserves them as artifacts for further investigations.
+
+```
+name: Run Integration Tests
+
+on:
+  push:
+    paths-ignore:
+      - 'README.md'
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
+  schedule:
+    # “At 00:00 on Sunday.”
+    - cron: "0 0 * * 0"
+  workflow_dispatch:
```

Comment:
not sure if it makes sense to keep all of this in the docs. it adds additional noise with no real value. 
If we keep it, we should at least explain what it does (adding inline comments would be sufficient).